src/analytics.js

```diff
@@ -0,0 +1,11 @@
+var _paq = _paq || [];
+_paq.push(['trackPageView']);
+_paq.push(['enableLinkTracking']);
+(function() {
+  var u="//stats.opkode.com/";
+  _paq.push(['setTrackerUrl', u+'piwik.php']);
+  _paq.push(['setSiteId', '1']);
+  var d=document, g=d.createElement('script'), s=d.getElementsByTagName('script')[0];
+  g.type='text/javascript'; g.async=true; g.defer=true; g.src=u+'piwik.js'; s.parentNode.insertBefore(g,s);
+})();
+
```

Comment:
Why are you adding such a script?

Reply:
These aren't mine - they're from here: https://conversejs.org/analytics.js and https://inverse.chat/inverse-analytics.js. Adding them to the source tree means it's clear they aren't proprietary.

Reply:
@gnunicode I don't think I want to add these to the repo. What then happens is that people have these scripts included in their self-hosted setups and then my analytics server gets (incorrectly) called by them.

Is there another way to do this?

Reply:
Other than that, thanks for the effort!

Reply:
I've removed the scripts. What could be useful is to add a comment to the analytics scripts stating the license (in the HTML file it says the analytics files are MPL-2.0 so a comment that states they are MPL'ed could be helpful). It's not needed for LibreJS to work as I've already stated the license of those files in the HTML files, but an analytics script with no license information on it can look like it's proprietary even if LibreJS lets it through.